test/integration/telemetry/test/index.test.js

```diff
@@ -90,7 +90,7 @@ describe('Telemetry CLI', () => {
   it('can disable telemetry with env NEXT_TELEMETRY_DISABLED', async () => {
     // next config is not reset between tests
     await runNextCommand(['telemetry', 'enable'])
-    const { stdout } = await runNextCommand(['telemetry', 'status'], {
+    const { stdout } = await runNextCommand(['telemetry'], {
```

Comment:
This change shouldn't be necessary, let's ensure this continues to work to avoid breaking change